protocol-configuration/proposalsConfig.ts

```diff
@@ -3,59 +3,59 @@ import tip_111 from '@proposals/description/tip_111';
 import tokemak_withdraw from '@proposals/description/tokemak_withdraw';
 import tip_112 from '@proposals/description/tip_112';
-import timelock_admin from '@proposals/description/timelock_admin';
+import tip_115 from '@proposals/description/tip_115';
 
 const proposals: TemplatedProposalsConfigMap = {
-  // tip_111: {
-  //   deploy: false, // deploy flag for whether to run deploy action during e2e tests or use mainnet state
-  //   totalValue: 0, // amount of ETH to send to DAO execution
-  //   proposal: tip_111, // full proposal file, imported from '@proposals/description/fip_xx.ts'
-  //   proposalId: '',
-  //   affectedContractSignoff: [
-  //     /*'dpiToDaiLBPSwapper',
-  //     'compoundDaiPCVDeposit',
-  //     'tribalCouncilSafe',
-  //     'ratioPCVControllerV2'*/
-  //   ],
-  //   deprecatedContractSignoff: [],
-  //   category: ProposalCategory.DAO
-  // },
-  // tip_112: {
-  //   deploy: false, // deploy flag for whether to run deploy action during e2e tests or use mainnet state
-  //   totalValue: 0, // amount of ETH to send to DAO execution
-  //   proposal: tip_112, // full proposal file, imported from '@proposals/description/fip_xx.ts'
-  //   proposalId: '',
-  //   affectedContractSignoff: ['core', 'fuseFixer', 'pcvGuardianNew'],
-  //   deprecatedContractSignoff: [],
-  //   category: ProposalCategory.DAO
-  // },
-  // tokemak_withdraw: {
-  //   deploy: false, // deploy flag for whether to run deploy action during e2e tests or use mainnet state
-  //   totalValue: 0, // amount of ETH to send to DAO execution
-  //   proposal: tokemak_withdraw, // full proposal file, imported from '@proposals/description/fip_xx.ts'
-  //   proposalId: '',
-  //   affectedContractSignoff: ['ethTokemakPCVDeposit'],
-  //   deprecatedContractSignoff: [],
-  //   category: ProposalCategory.TC
-  // },
-  timelock_admin: {
+  tip_111: {
     deploy: false, // deploy flag for whether to run deploy action during e2e tests or use mainnet state
     totalValue: 0, // amount of ETH to send to DAO execution
-    proposal: timelock_admin, // full proposal file, imported from '@proposals/description/fip_xx.ts'
+    proposal: tip_111, // full proposal file, imported from '@proposals/description/fip_xx.ts'
+    proposalId: '',
+    affectedContractSignoff: [
+      /*'dpiToDaiLBPSwapper',
+      'compoundDaiPCVDeposit',
+      'tribalCouncilSafe',
+      'ratioPCVControllerV2'*/
+    ],
+    deprecatedContractSignoff: [],
+    category: ProposalCategory.DAO
+  },
+  tip_112: {
+    deploy: false, // deploy flag for whether to run deploy action during e2e tests or use mainnet state
+    totalValue: 0, // amount of ETH to send to DAO execution
+    proposal: tip_112, // full proposal file, imported from '@proposals/description/fip_xx.ts'
+    proposalId: '',
+    affectedContractSignoff: ['core', 'fuseFixer', 'pcvGuardianNew'],
+    deprecatedContractSignoff: [],
+    category: ProposalCategory.DAO
+  },
+  tokemak_withdraw: {
```

Comment:
`tokemak_withdraw` can be removed as already executed on-chain - will speed up your e2es. You might need to update the block number to reflect that